components/AppBar/examples/AppBarDelegateForwardingExample.m

```diff
@@ -23,6 +23,8 @@
 
 @interface AppBarDelegateForwardingExample : UITableViewController
 @property(nonatomic, strong) MDCAppBar *appBar;
+@property(nonatomic, strong) MDCSemanticColorScheme *colorScheme;
```

Comment:
I know it's a pain in the butt in Objective-C, but we probably need to follow-up with an initialization of these properties to default values on each example.

Reply:
Please file a story to do as a follow-up and put at the bottom of the polish epic. Is not particularly critical.

Reply:
yaaawwww >.<